.github/workflows/ci-cpp.yaml

```diff
@@ -26,6 +26,9 @@ on:
     branches:
       - branch-*
 
+env:
+  MAVEN_OPTS: -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3
```

Comment:
shall we use 240 seconds as you written in your comment ?

Reply:
I was thinking about that. However I'm not exactly sure if the GitHub Flows environment is an Azure environment as referenced in WAGON-545.

The reason for the 25 second TTL is to ensure that it's shorter than any common firewall or NAT timeout. Some NATs have a 30 second idle timeout although that is very rare.

There shouldn't be harm in using the 25 second TTL since the connection pool will be able to pool connections well with a 25 second TTL. I don't see a reason to extend it to 240 seconds since there's not really a benefit.